### PR TITLE
Create blog app

### PR DESCRIPTION
**Pull request by SamTush**
## What does this PR do?

This PR is raised to implement the following features: 👍 :

- [x] Create a rails app
- [x] Use postgresql database
- [x] Build schema

## Description of the task to be completed ↓↓ 

- Create blog app

#### Thanks for reviewing my PR 😊

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[key_feature_1]**
```
Comment (rotimiazeez) [inline README.md]: Please ensure you add the key features of this project.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+- **[key_feature_2]**
+- **[key_feature_3]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+
+- [Live Demo Link](https://google.com)
```
Comment (rotimiazeez) [inline README.md]: Kindly just add coming soon or remove this section if you dont have your app deployed online.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+```sh
+
+```
+ -->
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 👥 Authors <a name="authors"></a>
```
Comment (rotimiazeez) [inline README.md:92]: This is a solo project, Kindly ensure only the right contributor is listed here 😀

File: README.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+
+- GitHub: [@githubhandle](https://github.com/githubhandle)
+- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[new_feature_1]**
```
Comment (rotimiazeez) [inline README.md]: Please ensure you list future features that can improve your application here. 🙏🏽

File: README.md [hunk 5]
```diff
@@ -0,0 +1,210 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
+
+If you like this project...
```
Comment (rotimiazeez) [inline README.md]: It will be nice to have a good support message here. 👍🏼

File: README.md [hunk 6]
```diff
@@ -0,0 +1,210 @@
...
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Write a message to encourage readers to support your project
```
Comment (rotimiazeez) [inline README.md]: Kindly remove all instruction text from your README 😉

File: README.md [hunk 7]
```diff
@@ -0,0 +1,210 @@
...
+
+If you like this project...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+I would like to thank...
```
Comment (rotimiazeez) [inline README.md]: Your acknowledgement seem incomplete, Kindly fix it. 👌🏽

File: README.md [hunk 8]
```diff
@@ -0,0 +1,210 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
```
Comment (rotimiazeez) [inline README.md:129]: Kindly add a valid MIT License link here 😊

**Review by rotimiazeez — COMMENTED** [review]
# Hi @SamTush, 

### While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project have no professional README file. Kindly make the necessary changes as required 🙏🏽 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🍾✔🎉**
Hello @SamTush 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- Descriptive PR.
- Good commit messages.
- Correct git-flow.

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**